src/js/utils.js

```diff
@@ -41,3 +41,7 @@ export function decreaseAllMemoIndexes() {
     memo.style.zIndex = --index;
   }
 };
+
+export function getCurrentWorkspace() {
+  return getLocalStorageItem("manifest_workspace") || setLocalStorageItem("manifest_workspace", "0");
```

Comment:
I opted for 0 - 9 being the keys but we could use whatever you think makes more sense, I also made 0 being the new default workspace